### PR TITLE
LPD-27537 Guest user cannot view Knowledge Base Article through struts action /knowledge_base/find_kb_article

### PR DESCRIPTION
**Pull request by jonathanmccann**
# What is this trying to solve?

https://liferay.atlassian.net/browse/LPD-27537

Guest user cannot view Knowledge Base Article through struts action /knowledge_base/find_kb_article

# How am I fixing it?

These changes add `/knowledge_base/find_kb_article` to the public path so authentication is not required.

This path was previously a public path when Knowledge Base was still a plugin (https://github.com/liferay/liferay-plugins/blob/6.2.x/portlets/knowledge-base-portlet/docroot/WEB-INF/src/portal.properties#L14-L15) and seems to have been lost during the migration to an OSGi module.

If this was done purposefully and the path should require authentication please let me know. Additionally, any feedback on the Playwright test would be appreciated.

# How can you verify that it works?

Run `npm run test -- -g "LPD-27537"`

To test manually please see the steps on https://liferay.atlassian.net/browse/LPD-27537

### CODE REVIEW

**Comment by jonathanmccann** [comment]
ci:test:relevant

**Review by ambrinchaudhary — APPROVED** [review]
Playwright test LGTM

File: modules/test/playwright/tests/knowledge-base-web/knowledge-base-web.spec.ts [hunk 1]
```diff
@@ -10,31 +10,63 @@ import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
...
+				title,
+			});
+
+		await performLogout(page);
+
+		await page.goto(
+			liferayConfig.environment.baseUrl +
+				'/c/knowledge_base/find_kb_article?resourcePrimKey=' +
+				knowledgeBaseArticle.id
+		);
```
Comment (marcogalluzzi) [inline modules/test/playwright/tests/knowledge-base-web/knowledge-base-web.spec.ts:62]: Can you please add the url to the util class `KnowledgeBaseUrls` ? Thanks

**Review by marcogalluzzi — COMMENTED** [review]
The code looks good to me. The only doubt I have is the same as you have: can we make the path public again? Please @adolfopa have a look

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:report:11433902

**Comment by ambrinchaudhary** [comment]
> Unique Failures
> NONE,N/A,Unique,N/A
Result of the report. I force forward it.

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Pull request closed without merging**